ui/src/components/stepper/QasStepper.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="qas-stepper" :class="classes">
+    <q-stepper ref="stepper" v-model="model" active-color="primary" active-icon="none" animated :contracted="$qas.screen.untilLarge" done-color="primary" done-icon="none" flat :header-class="headerClass" inactive-color="grey-6" keep-alive>
```

Comment:
"active-icon" é "none" mesmo?

Reply:
sim, isso é um caso onde qd o for o stepper ativo, ele altera o ícone